src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -41,7 +41,7 @@ public sealed partial class FileCodeModel : AbstractCodeModelObject, EnvDTE.File
             ITextManagerAdapter textManagerAdapter)
         {
             // Keep track that code model was accessed.  We want to get a sense of how widespread usage of it still is.
-            Logger.Log(FunctionId.CodeModel_FileCodeModel_Create);
+            Logger.LogDebug(FunctionId.CodeModel_FileCodeModel_Create);
```

Comment:
will this make it so that htis information is actually logged to telemetry?

Reply:
No. Should Debug be logged to telemetry? Or are you worried about this specific instance. In general, I would consider Debug as 

> Logs that are used for interactive investigation during development. These logs should primarily contain information useful for debugging and have no long-term value.

Information and higher would be telemetry.

Reply:
My presumption is that all, or nearly all, of these log methods were intended to actually get us teh telemetry to make decisions :)

Reply:
Then how do you log to ETL, log files, etc? Verbosity is important here. That seems like a wild presumption that anything that says "Log" is telemetry. We definitely can make it that way, but that seems bizarre and unlike anything I've used elsewhere.

Reply:
So i've never once intended to log to ETL/log-files/etc.  Because the logging i've put in is to understand what's going on on user machines :)

Effectively, if there's an issue internally, we just repro or collect trace from internal person.  So the need for the logging is to understand what's going on externally.

I agree it's likely not all one or the other.  But i'm not really understanding the value in logging that isn't happening remotely :)

Happy to discuss this in the design meeting and see how everyone feels here.

Reply:
Sounds good. If consensus is that we should be always sending, then we can change to that. Let's bring it up on Monday :)

Reply:
I totally forgot I'm OOF. Feel free to get a general consensus on thinking though, or we can chat next week about it.